axelrod/_strategy_utils.py

```diff
@@ -45,11 +45,19 @@ def limited_simulate_play(player_1, player_2, h1):
     player_2's strategy method to set any internal variables as needed. If you
     need a more complete simulation, see `simulate_play` in player.py. This
     function is specifically designed for the needs of MindReader."""
-    h2 = player_2.strategy(player_1)
+    h2 = cheating_bastard(player_1, player_2)
     update_history(player_1, h1)
     update_history(player_2, h2)
 
 
+def cheating_bastard(cheater, opponent):
```

Comment:
no problem.  i'll put some work in making this more presentable, make tests for the simulated_strategies, and some tests for cheating_bastard (are you sure that name needs changing? ;) ).  

names:

- `inspect_strategy(inspector, opponent):` 
- `foil_strategy_inspections(self, opponent):`

how do those sound?